src/lib/constants.js

```diff
@@ -102,7 +102,7 @@ export const PDFJS_WIDTH_PADDING_PX = 40; // Should match SCROLLBAR_PADDING in p
 export const PDFJS_HEIGHT_PADDING_PX = 5; // Should match VERTICAL_PADDING in pdf_viewer.js
 
 export const DOC_LEGACY_STATIC_ASSETS_VERSION = '2.76.0'; // IE11 only
-export const DOC_STATIC_ASSETS_VERSION = '2.90.0'; // Modern browsers
+export const DOC_STATIC_ASSETS_VERSION = '2.93.0'; // Modern browsers
```

Comment:
Can we move the Cypress upgrade to its own PR?

Reply:
I'll move this pdfjs upgrade to its own PR and keep the cypress upgrade here since it changed so many files